docs/application/dotnet/guides/security/requesting-permissions.md

```diff
@@ -141,6 +141,89 @@ To check whether an application has permission to use a privilege, and to reques
 
     If the decision is definitive, any subsequent `RequestPermission()` calls result in an immediate response with an appropriate result: `AllowForever` or `DenyForever`. However, the user can change the status of privacy-related privileges later by modifying the privacy settings on the device. For this reason, the application must always check the status of privacy-related privileges before using protected functionality.
 
+## Requesting Multiple Permissions
+
+This section describes how to check and request multiple privileges in a single API call.
+
+> **Note**
+>
+> Multiple privileges in a single API call are supported from Tizen 5.5.
+
+To check whether an application has permission to use a privilege, and to request permission if required:
+
+1. To verify whether an application has permission to use privileges, use `CheckPermissions()`:
+```csharp
+    string[] privileges = new [] {"http://tizen.org/privilege/account.read",
+                                  "http://tizen.org/privilege/alarm"};
+    CheckResult[] results = PrivacyPrivilegeManager.CheckPermissions(privileges).ToArray();
+```
+The results of the call is stored in `CheckResult` array.
+
+2. React to the permissions check appropriately:
+
+  ```csharp
+    List<string> privilegesWithAskStatus = new List<string>();
+    try {
+        for (int iterator = 0; iterator &lt; results.Length; ++iterator)
+        {
+  ```
+ -   If the result value is `Allow`, the application is allowed to perform operations related to the privilege. For example, the application can enable additional UI elements or functionalities.
+  ```csharp
+            switch (results[iterator])
+            {
+                case CheckResult.Allow:
+                    // Privilege can be used
+                    break;
+  ```
+
+  -   If the result value is `Deny`, the application is not allowed to perform operations related to the privilege. Any attempt to use such functionality without the user's consent fails. Usually, this means that invoking any method that involves the privilege results in an error.
+  ```csharp
+                case CheckResult.Deny:
+                    // Privilege can't be used
+                    break;
+  ```
+
+  -   If the result value is `Ask`, the application must request permission from the user with the `RequestPermissions()` method, which displays a dialog box. When the user makes a decision, the answers are returned as
+  [Tizen.Security.RequestMultipleResponseEventArgs](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Security.RequestMultipleResponseEventArgs.html).
+
+```csharp
```

Comment:
Please fix the alignment for the code block.

Reply:
done